node/silkworm/downloader/header_downloader.cpp

```diff
@@ -31,7 +32,10 @@
 namespace silkworm {
 
 HeaderDownloader::HeaderDownloader(SentryClient& sentry, Db::ReadWriteAccess db_access, ChainIdentity chain_identity)
```

Comment:
should `SentryClient& sentry` be passed as a const.

Reply:
SentryClient will be completely refactored in the next PR